back/app/services/admin_publications_filtering_service.rb

```diff
@@ -41,11 +41,25 @@ def for_homepage_filter(scope)
     projects = Project.where(id: scope.where(publication_type: Project.name).select(:publication_id))
     filtered_projects = ProjectsFilteringService.new.filter(projects, options)
 
+    if options[:search].present?
+      filtered_projects = filtered_projects.search_by_all(options[:search])
+    end
```

Comment:
```suggestion
    filtered_projects = filtered_projects.search_by_all(options[:search]) if options[:search].present?
```
FYI: Rubocop will let you do this (line is not too long), but your original is clearer.

Reply:
Haha I wrote it this way after reading the original and being surprised when I got to the end of the line and it was an `if` statement. I think for smaller conditions the trailing `if` is fine but in this case there's a decent amount of logic before it, so IMO it makes sense to be a little more verbose and make it clear up front what the conditional is